<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="4738" started-at="2018-01-23T14:32:10Z" finished-at="2018-01-23T14:32:14Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="4738" started-at="2018-01-23T14:32:10Z" finished-at="2018-01-23T14:32:14Z">
      <class name="Test.PracticeOne.Maven_Practice1.MySelenium">
        <test-method status="PASS" signature="BeforeClass()[pri:0, instance:Test.PracticeOne.Maven_Practice1.MySelenium@bbfa04]" name="BeforeClass" is-config="true" duration-ms="4682" started-at="2018-01-23T20:02:10Z" finished-at="2018-01-23T20:02:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeClass -->
        <test-method status="FAIL" signature="TestCase1(java.lang.String)[pri:0, instance:Test.PracticeOne.Maven_Practice1.MySelenium@bbfa04]" name="TestCase1" duration-ms="0" started-at="2018-01-23T20:02:14Z" finished-at="2018-01-23T20:02:14Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Cannot inject @Test annotated Method [TestCase1] with [class java.lang.String].
For more information on native dependency injection please refer to http://testng.org/doc/documentation-main.html#native-dependency-injection]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Cannot inject @Test annotated Method [TestCase1] with [class java.lang.String].
For more information on native dependency injection please refer to http://testng.org/doc/documentation-main.html#native-dependency-injection
	at org.testng.internal.Parameters.checkParameterTypes(Parameters.java:334)
	at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:274)
	at org.testng.internal.Parameters.createParameters(Parameters.java:562)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:672)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1316)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1044)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1134)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestCase1 -->
        <test-method status="PASS" signature="AfterClass()[pri:0, instance:Test.PracticeOne.Maven_Practice1.MySelenium@bbfa04]" name="AfterClass" is-config="true" duration-ms="0" started-at="2018-01-23T20:02:14Z" finished-at="2018-01-23T20:02:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterClass -->
      </class> <!-- Test.PracticeOne.Maven_Practice1.MySelenium -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
